examples/simple-go/go.mod

```diff
@@ -3,11 +3,13 @@ module simple-go
 go 1.21
 
 require (
-	github.com/pulumi/pulumi-aws-apigateway/sdk/v2 v2.0.0
+	github.com/pulumi/pulumi-aws-apigateway/sdk/v2 v2.2.1
 	github.com/pulumi/pulumi-aws/sdk/v6 v6.5.0
 	github.com/pulumi/pulumi/sdk/v3 v3.94.2
 )
 
+replace github.com/pulumi/pulumi-aws-apigateway/sdk/v2 => ../../sdk/
+
```

Comment:
Whoops, glad we're actually testing the right version now!

Reply:
Ah, didn't mean to leave that in. We actually tested the right version because ProgramTest injects this `replace` at run time. I added it for manual testing. It doesn't hurt, though, I think.

Reply:
Nah, I'll take it out to be safe.